nginx/prod.nginx.conf

```
@@ -6,7 +6,7 @@ upstream api {
   server api:8081 weight=10 max_fails=3 fail_timeout=30s;
 }
 
-proxy_cache_path /nginx-cache levels=1:2 keys_zone=my_cache:30m inactive=60m use_temp_path=off max_size=4g;
+proxy_cache_path /nginx-cache levels=2:2:2 keys_zone=my_cache:300m inactive=1y use_temp_path=off max_size=4g;
```

If `inactivte=1y` and `max_size=4g`, does it mean when the size of cache exceeds 4 GB, no data will be cached? If so, it would be better to cache recent visits than to keep very old visits in the cache.

---

How about 10g? It's not enough if all the cache paths are used, but it should still be plenty:
```suggestion
proxy_cache_path /nginx-cache levels=2:2:2 keys_zone=my_cache:300m inactive=1y use_temp_path=off max_size=10g;
```
I'd rather keep the `1y` unless the storage limit is hit.

---

@nanjiangshu what do you think?

---

OK. I think it is fair enough to set max_size to 10g since the root volume of VM is usually not very big. 
I will approve this PR